settings.py

```diff
@@ -19,5 +19,5 @@ def get_key(key_name):
 PRIVATE_KEY_PASSWORD = os.getenv("PRIVATE_KEY_PASSWORD", "digitaleq")
 
 LOGGING_FORMAT = "%(asctime)s|%(levelname)s: %(message)s"
-LOGGING_LOCATION = "error.log"
+LOGGING_LOCATION = "logs/error.log"
```

Comment:
Probably should loose the name error.log (maybe decrypt.log instead), given it covers all types of logging at the level which has been set.